stellar/deploy-contract.js

```diff
@@ -207,8 +151,8 @@ async function deploy(options, config, chain, contractName) {
         return;
     }
 
-    const wasmPath = await getWasmPath(options, contractName);
-    const wasmHash = await uploadWasm(wasmPath, wallet, chain);
+    const wasmResolvedPath = await getWasmFilePath(options, contractName);
+    const wasmHash = await uploadWasm(wasmResolvedPath, wallet, chain);
```

Comment:
Could you reference the latest code from main?
```suggestion
    const wasmHash = await uploadContract(contractName, options, wallet, chain);
```

`uploadContract` function should be updated:
```
async function uploadContract(contractName, options, wallet, chain) {
    const wasmFile = await getWasmFile(options, contractName);
    return await uploadWasm(wasmFile, wallet, chain);
}
```

Reply:
in this case i think that the updates in this PR are better since we are moving the file path determination to a prehook and its already handled so the extra logic of the function uploadContract is not needed and i can call uploadWasm directly